Tests/DashboardTest.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using HotelPriceScout.Data.Model;
+using HotelPriceScout.Pages;
+using HotelPriceScout.Data.Interface;
+
+
```

Comment:
whats up with the empty space, der er så meget empty space her at selv sarmi ikke ville ramme noget kode.

Reply:
Skal nok slette det :(((